kurlkinds/Makefile

```diff
@@ -58,7 +59,7 @@ test-crd:
 
 # get/update controller-gen
 update-controller-gen:
-	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.2
+	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0
```

Comment:
hmmm. why did this revert?

Reply:
this was the version pinned in the troubleshoot spec.